app/src/main/java/com/example/bigowlapp/utils/NotificationListenerManager.java

```diff
@@ -75,6 +75,10 @@ private void handleNotificationRequest(Notification notification, Context contex
         String channel = notification.getTitle();
         createNotificationChannel(context, channel);
 
+        notification.setUsed(true);
+        repositoryFacade.getCurrentUserNotificationRepository()
+                .updateDocument(notification.getUid(), notification);
+
```

Comment:
Please remove, this is already happening in lines  84-86. Will cause double notifications?